include/silo_api/request_handler.h

```diff
@@ -6,8 +6,17 @@
 #include <Poco/Net/HTTPRequestHandler.h>
 #include <Poco/Net/HTTPServerRequest.h>
 #include <Poco/Net/HTTPServerResponse.h>
+#include <nlohmann/json.hpp>
 
 namespace silo_api {
+
+struct ErrorResponse {
+   std::string error;
+   std::string message;
+};
+
+NLOHMANN_DEFINE_TYPE_NON_INTRUSIVE(ErrorResponse, error, message);
```

Comment:
Doesn't work, because the `InfoHandler` needs to know that the generated functions exist. The alternative would be to only declare the functions here and include the macro in the .cpp file, but that results in more code...

Reply:
If there will be more, we can consider doing this. For now I would suggest leaving it like this.